src/Caching/Hybrid/src/Microsoft.Extensions.Caching.Hybrid.csproj

```diff
@@ -9,7 +9,7 @@
         net 5.0 : [DynamicallyAccessedMembers], EncodingExtensions.GetString/GetBytes, IsExternalInit
         net 7.0 : ArgumentNullException.ThrowIfNull
     -->
-    <TargetFrameworks>$(DefaultNetCoreTargetFramework);$(DefaultNetFxTargetFramework);netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);$(DefaultNetFxTargetFramework);netstandard2.0;netstandard2.1;$(CurrentLtsTargetFramework)</TargetFrameworks>
```

Comment:
Is the plan for this project to _always_ support the current TFM + the most recent LTS TFM, or should this change be undone when we go to `net10.0`?

Reply:
We can leave this alone - for LTS builds the `DefaultNetCoreTargetFramework` and `CurrentLtsTargetFramework` will match, meaning it gets de-dup'd and we just get the LTS target (plus the spares)